### ISSUE TITLE
Recognition for volunteer outside Reps

### ISSUE THREAD

**Issue by kelimuttu** [mozilla/Reps#237]
## Goal: 
Provide an official recognition system within the Reps for volunteer outside the program.

## Related Issues & Links: 
[Reference meta issues and past issues] #

## Roles:

Responsible: @couci 

Accountable: @couci 

Supporting: @kelimuttu

## Required: 
- [ ] Bring the topic to the Council meeting 
- [ ] A new recognition system has been proposed
- [ ] The proposed system has been approved by the council
- [ ] Communication to the rest of the Reps

**Comment by Mte90** [comment]
Uhm very interesting!
I think that we have a lot to discuss for that!

**Comment by kelimuttu** [comment]
@Mte90 feel free to write it down here if you have any suggestions or ideas in mind..

**Comment by Mte90** [comment]
I think that is something to define with the other programs that can have their way or solution for that so will be easily to create conflict.
So my idea for us will be for mozillians (that have the profile and also at least a vouch) that joined an event (or organized one with the help of a rep, rep have their way of recognize: ROM, the letter (only on 2015)).
Next we need to find a way to list their activities because a submission system from other volunteers can be a mess to manage.

So probably the best way to suggest is from the Rep itself but we need to define a way because the mozillians not track what they do.

**Comment by Faisalaziz** [comment]
I believe we can have some inputs from Alex lakatos on this as he was engaged in recognition and had some words with foundation people.
I am too interested in this topic and would like to provide some suggestions on this.
•	Initiation
o	Recognition can be initiated by Reps, Staff or a cohort members
o	Could be a Privilege for filing self-recognition.
•	Filing 
o	Need a template describing why an individual have to be recognized.
o	Factors like adherence to current goals would be considered on priority.
o	Documentation, blogs, pictures links could be vital in this process.
o	For functional recognition links to contribution dashboards needs to be included if not available, functional lead vouch is required.
•	Screening
o	Screening part can be executed by Reps Review Team/Council.
o	Monthly recognition is manageable just like ROM.
o	If we get too many requests then we should opt for vote, however I believe we should cater a good number for recognition since count of  non-reps will be on higher side.
o	Review Team/Council will finalize people and post like ROM.
•	Recognition ways:
o	Blog Post with Pictures like ROM.
o	Special Swag 
o	Badge 
o	Mention on vouch section Mozillians profile or somewhere  on Mozillian profile.
o	Invitation to join Reps Program for outstanding contributors.
o	Wall of Fame
o	Certificates
•	Challenges
o	Recognition process should simple and easy as most of the people would drop an idea to recognize a fellow Mozillian because the process is too complicated or time consuming.
o	Minimal form submission often preferred.

**Comment by kelimuttu** [comment]
@Mte90 Totally makes sense. The only thing is that we need a good infrastructure to support that idea (something that we're lacking of right now). 

@Faisalaziz Thanks for your suggestion. Totally agree with your proposed steps. And I think the challenge would be how to create a unified mechanism so we can avoid people who has power to recognize only the people they want.

**Comment by Faisalaziz** [comment]
@kelimuttu 

> the challenge would be how to create a unified mechanism so we can avoid people who has power to recognize only the people they want.

That is why i proposed "Self Nomination" and for screening "Review Team/Council". There wont be any "people with power" , Any individual who has done contribution can nominate himself/herself directly with proper proofs/facts and will be screened by Review Team/Council. Thats how we avoid conflicts and i am sure contributors are much more satisfied if decision is coming from review team /or council even in case of rejection.

**Comment by kelimuttu** [comment]
@Faisalaziz You're absolutely right about the screening. Totally agree

**Comment by emmairwin** [comment]
also self nomination skews to those most confident in their ability, and in speaking up for themselves which rewards a specific type of person.  
We need less self-reporting, and more surfacing the accomplishments of others. OR we need clear criteria for recognition, including what it isn't .   

Additionally, real-world-value is a critical issue to solve for in recognition.  What is of value might be different from region to region but I think we need to ask people 'what would you most value for your resume? where would that live?  What format would you want that in?  

we have tried many of the things listed above, so also my question is how do you implement standards that are true for everyone, that are not bottle necked by staff/council, how do you aim for consistency, predictability (milestones, timelines) all of these things are attributes of recognition.

**Comment by emmairwin** [comment]
Might be interesting to look at projects like freecodecamp that work use certificates for specific types of recognition . it's coding focused (front end , back end) but there's probably something to learn :   https://www.freecodecamp.org/challenges/claim-your-front-end-development-certificate

reaching out to ask them how this has gone (honest vrs dishonest applications, diversity of recipients etc might be interesting).  From their site on certificates awarded :

    Front End: 6,654earned
    Data Viz: 868earned
    Back End: 578earned
    All three: 371earned

**Comment by Mte90** [comment]
The certification system was often discussed I agree that can be the best solutions because with Reps portal it is possible for us to know what they are doing (and suggest them to report what they are doing).
But not all the mozillians are Reps so that's the problem. Community analytics can help us to show the status on specific kind of projects but in not all of them (like sumo or l10n).

So probably we need to evaluate the best way to enumerate in points what Mozillians are doing. After that we can do the measuring solutions for every kind of activities and release certificates or whatever.

**Comment by emmairwin** [comment]
This is something we're thinking about as well for education/learning modules. How do we know people completed what we asked.  It might be easier to ask people to complete an assessment(do a small thing) or demonstrate knowledge(complete an quiz).  Perhaps instead of setting a bar high to track progress in systems, it's easier to outline ways they can demonstrate competency gain through an experience.  That would also help non-coding mozillians who contribute a lot, but never turn up on metrics dashboards.

Another thing to think about is why badges (which are really just digital certificates) lost value in community.  I have a few opinions, but will leave it there.

**Comment by kelimuttu** [comment]
That's a really good point, Emma. There's seems like a need on checking with the community what's their ideal recognition would be like so we can have a baseline data to move forward.

**Comment by Mte90** [comment]
Maybe we can do a survey with specific questions about it?

**Comment by kelimuttu** [comment]
That seems like an idea. Maybe you can start through having discussion about it in the reps council meeting?

**Comment by Mte90** [comment]
I will add in the notes for the next meeting :-)

**Comment by emmairwin** [comment]
@kelimuttu yes, it would be good to ask them - but also challenge them to think outside of  established ways we believe recognition should work. 

I found when we ask this question, people tend to think about it from a community mangers perspective (about others, and that they are helping me brainstorm for others), when we really need people to think only about themselves for the moment.  What would that be that would help you in your career? In your school?  What have you seen elsewhere that you thought sounded like it would be valuable to put on a resume?  (just brainstorming). 

I also think it would be good to say ask reverse questions - what hasn't worked?  Why was that?  


Happy to help here

**Comment by jgmac1106** [comment]
@emmairwin When you reach a specific level you can ask for and usually receive specific recognition. Though this is not immune to being more available to those who advocate as your previously note.

However if I have ever needed a letter of recommendation or an introduction I can ask privately and get this support from other contributors, both on and off the payroll.

Make these normal every day offerings transparent. Maybe just add buttons like, "Request a letter of recommendation, Request a LinkedIn endorsement, etc." These could be triggered by a number of vouches or staff initiated.

I know  everyone is not on the open badges train (I consider myself a recently converted naysayer) but the changes to the standards in 2.0, specifically the endorsement class, could have potential for Mozilla.

I think the badge effort never worked because neither the earners (contributors) nor their audience (potential employers or schools) did not see any value in it nor had no idea what people were talking about. 

If different areas of Mozilla developed pathways and the endorsement levels increased there is a future in open recognition.

**Comment by kelimuttu** [comment]
@emmairwin That's definitely true. That would be super exciting to hear what they think about it. From what I see, there is a need to categorize the mechanism into 2 type of recognition. From top to bottom, and from bottom to top. Or even one more from volunteer to the other volunteer. In that way, we can know exactly how we need to design the flow of the system and therefore explore the recognition in each category.

**Comment by emmairwin** [comment]
Thanks @jgmac1106 for me the failures of badges were:

1) we gave them away(and others do this still) for various levels of accomplishment - a badge could be for something major, or something trivial.(no standards)
2) we gave badges to people that they didn't necessarily care about, which made them feel unimportant. 
3) We gave badges away for things people did care about, but servers holding the metadata (image etc) went away, and so people essentially LOST their badges. (speaking from experience).
4) Badges could not be say, imported to LinkedIN, or other professional platforms related to CVs . (although yes, backpack allowed you to pull them into your own site.

I still like the idea of badges, but it's the baggage of the past that's hard to overcome.(no offense to good work of badges people)
